readme.md

```diff
@@ -20,19 +20,22 @@ Please report all hotpatch-related problems to them :)
 First, we need to install a specific version of the Dioxus CLI.
 
 ```sh
-cargo install dioxus-cli --git https://github.com/DioxusLabs/dioxus --rev b2bd1f
+cargo install dioxus-cli --git https://github.com/laundmo/dioxus --branch=subsecond-fuse-ld
 ```
 
 Depending on your OS, you'll have to set up your environment a bit more:
 
+#### Windows
```

Comment:
Any reason this can't still be a `details`? All users will only care about 1/3 of the information presented in the setup chapter otherwise

Reply:
No, except that `details` are quite small, easy to miss when quickly scrolling, and require clicking on them just to see 'should (maybe) just work'

Theres no need to reduce the amount of text there, it just makes what people care about harder to reach.